src/os/bluestore/BlueStore.cc

```diff
@@ -5002,6 +5024,8 @@ int BlueStore::_mount(bool kv_only)
   if (r < 0)
     goto out_alloc;
 
+  open_statfs();
+
```

Comment:
need the same thing in fsck().  Or better yet just put it in _open_super_meta().